### ISSUE TITLE
[Bug] Unable to download generated CSS with the "Export Dynamic Theme" option

### ISSUE THREAD

**Issue by MPeti1** [darkreader/darkreader#6555]
# Bug Report

## Bug Description
Clicking the `Export Dynamic Theme` button in the `Settings -> Manage Settings` menu (which is visible after switching to the new design of the extension) results in no action on certain websites, but not all.
It looks like the extension does not log errors to the browser console when it fails, but I think it starts the process, because when I click on the mentioned button, the CPU usage of Firefox grows for a few seconds. Also, if I click the button many times, Firefox will notify me multiple times (with a zop notification shown just for a split second) that Dark Reader might be dlowing down the browser (which I take as verification that the extension starts doing the hard work)

## Steps To Reproduce

1. Go to [is there a way to only disclose the link to the developers?]
2. Open Dark Reader's menu with the toolbar icon
3. Click the `Dev tools` button
4. In the new window, click `Preview new design`
5. Open Dark Reader's menu with the toolbar icon
6. Open the `Settings` menu
7. Open the `Manage settings` menu
8. Click `Export Dynamic Theme`, to receive a .css file as a download

## Expected Behavior

I expect a CSS file, that includes all of the styling that Dark Reader applies to the page when it turns it into dark colors.

## Actual Behavior

No css file is generated.

## System Information

- OS:  Windows
- Browser:  Firefox 88.0.1
- Dark Reader Version:  4.9.34

## Additional Context

The website for which I'm unable to export a dynamic theme is allowed for the extension to be themed, and it is set to dynamic mode. I think the function worked a few months earlier for the same website.

**Comment by Gusted** [comment]
Hmm, most likely it's some kind of large CSS, which we currently are formatting and then export it to the user, the formatting can be a pretty heavy resource task, it will be disabled for large CSS with #5697.

**Comment by MPeti1** [comment]
Do you mean that the export does not happen because the extension is still working on darkening the page?

**Comment by Gusted** [comment]
No, the code to format the CSS to a nice output is doing a lot of hard work which can cause that it won't be able to output a file.

**Comment by MPeti1** [comment]
Hmm, but the page is not huge. By the looks it seems to be pretty simple.
Also, according to the style editor tab of the browser dev tools, it has less than a thosand style rules
![image](https://user-images.githubusercontent.com/9215491/130319542-ae2d5270-cfed-4002-a4fe-a5f2017beb13.png)

**Comment by Gusted** [comment]
The firefox's Style Editor tab is a big joke that doesn't even work 9/10 times. Their is more likely a lot more styles/code then being showed. Also if it showed the "Dark reader is slowing this website" bar, it definitely means that it's doing something heavy(most likely a lot of CSS lines).
